tests/sentry/issues/test_escalating.py

```diff
@@ -21,6 +23,8 @@
 from sentry.utils.cache import cache
 from sentry.utils.snuba import to_start_of_hour
 
+TIME_YESTERDAY = (datetime.now() - timedelta(hours=24)).replace(hour=6)
```

Comment:
nit: you can do `timedelta(hours=24)` to make this a bit cleaner, but it's up to you.
thanks for taking this on!